draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
This seems to break the implicit stream opening feature that we currently have, because if the maximum stream open is X and then you get a STREAM frame for Y >> X, you don't know if the other side is going to open W where X < W < Y unidirectional or bidirectional